docs/Serial.md

```diff
@@ -174,24 +174,27 @@ Note: Also has a boolean AUTOBAUD. It is recommended to use a fixed baudrate. Co
 
 The Serial Port baudrates are defined as follows:
 
-| ID | Baudrate  |
-| -- | --------- |
-| 0  | Auto      |
-| 1  | 9600      |
-| 2  | 19200     |
-| 3  | 38400     |
-| 4  | 57600     |
-| 5  | 115200    |
-| 6  | 230400    |
-| 7  | 250000    |
-| 8  | 400000    |
-| 9  | 460800    |
-| 10 | 500000    |
-| 11 | 921600    |
-| 12 | 1000000   |
-| 13 | 1500000   |
-| 14 | 2000000   |
-| 15 | 2470000   |
+| ID | Baudrate   |
+| -- | ---------- |
+| 0  | Auto       |
+| 1  | 9600       |
+| 2  | 19200      |
+| 3  | 38400      |
+| 4  | 57600      |
+| 5  | 115200     |
+| 6  | 230400     |
+| 7  | 250000     |
+| 8  | 400000     |
+| 9  | 460800     |
+| 10 | 500000     |
+| 11 | 921600     |
+| 12 | 1000000    |
+| 13 | 1500000    |
+| 14 | 2000000    |
+| 15 | 2470000    |
+| 16 | BAUD_COUNT |
```

Comment:
Not sure this should be in here - this is merely a 'stop element' that is used programmatically, but it isn't intended to be settable by the user, or exposed in any other external way.

Reply:
Makes sense. Will close the PR.